packages/components/molecules/f-restaurant-card/src/components/subcomponents/DeliveryTimeMeta/DeliveryTimeMeta.vue

```diff
@@ -72,6 +72,17 @@ export default {
 
 <style lang="scss" module>
 .c-deliveryTimeMeta {
-    margin-right: spacing(b);
+    display: flex;
+
+    &:after {
+        content: '\2022'; // round bullet character
+        color: $color-content-subdued;
+        margin: 0 spacing(a);
+        font-weight: $font-weight-regular;
```

Comment:
I wonder if we can move it out into a mixin, as it's duplicated in a couple of places, and it represents a defined element "dot separator"